.github/workflows/test.yaml

```
@@ -15,7 +15,7 @@ jobs:
       TOXENV: "alldeps-withcov-posix,coverage-prepare,codecov-push,coveralls-push"
     strategy:
       matrix:
-          python-version: ['3.5.4', 3.5, 3.6, 3.7, 3.8, 3.9, pypy-3.6, pypy-3.7]
+          python-version: ["3.6.7", 3.6, 3.7, 3.8, 3.9, pypy-3.6, pypy-3.7]
```

Maybe add a comment for others to understand what we are doing here:
```suggestion
          # Run on the minimum micro Python version that we can get on CI.
          # When updating the minimum Python version here, also update the
          # `python_requires` from `setup.cfg`.
          # Run on latest minor release of each major python version.
          python-version: ["3.6.7", 3.6, 3.7, 3.8, 3.9, pypy-3.6, pypy-3.7]
```